packages/cosmos_ui_components/lib/components/cosmos_bottom_sheet_header.dart

```diff
@@ -15,9 +15,7 @@ class CosmosBottomSheetHeader extends StatelessWidget {
     return Column(
       mainAxisSize: MainAxisSize.min,
       children: [
-        const SizedBox(
-          height: 40,
-        ),
+        SizedBox(height: CosmosTheme.of(context).spacingM),
```

Comment:
bottomSheetHeader is meant to be this widget, thus you can make it accept the `leading` and `actions` parameters as well as `titleTextStyle` and keep the design consistent between the state without any actions and with actions.
![CleanShot 2021-11-24 at 18 16 56@2x](https://user-images.githubusercontent.com/1711474/143285423-e6ac47a9-4198-4c2a-b1cb-b7628b3d28d4.png)
:

Reply:
Updated